CONTRIBUTING.md

```diff
@@ -3,7 +3,8 @@
 ## Contribution Steps
 
 * [Build and debug the extension](Documentation/Building%20the%20Extension.md).
-* File an [issue](https://github.com/Microsoft/vscode-cpptools/issues) and a [pull request](https://github.com/Microsoft/vscode-cpptools/pulls) with the change and we will review it.
+* Before filling a new issue **check if the issue already exists in it.** 
```

Comment:
I think "in it" should be removed. I don't understand what "it" is referring to.